server/channels/api4/system.go

```diff
@@ -55,6 +56,7 @@ func (api *API) InitSystem() {
 
 	api.BaseRoutes.APIRoot.Handle("/analytics/old", api.APISessionRequired(getAnalytics)).Methods("GET")
 	api.BaseRoutes.APIRoot.Handle("/latest_version", api.APISessionRequired(getLatestVersion)).Methods("GET")
+	api.BaseRoutes.APIRoot.Handle("/root_check", api.APISessionRequired(checkForRootUser)).Methods("GET")
```

Comment:
An additional API endpoint is a bit too much. Let's add the `is_root` flag to the `/ping` endpoint.

Reply:
Yes, makes sense. Working on it.